infra/terraform/test-org/org/locals.tf

```diff
@@ -624,7 +624,7 @@ locals {
       name            = "terraform-google-analytics-lakehouse"
       org             = "GoogleCloudPlatform"
       description     = "Deploys a Lakehouse Architecture Solution"
-      owners          = ["jasondavenport"]
+      owners          = ["davenportjw"]
       topics          = local.common_topics.da
       groups          = [local.jss_common_group]
```

Comment:
[not action needed]
Just a heads-up: you can also specify ownership by referring to a GitHub team using the `groups` field — in case you'd like to extend ownership to your entire team. :)
Example: https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/1543